robot/rover/Science/Science.ino

```diff
@@ -711,13 +841,21 @@ int drill_speed(int input_drill_speed) {
 }
 
 float elevator_feed(int input_elevator_feed) {
-  if (input_elevator_feed < 0) {
-    input_elevator_feed = 0;
+  if (input_elevator_feed < 15) { //Below 15% the equation is not reliable
+    input_elevator_feed = 0;     //and the motor has a tendancy to stall
   }
   if (input_elevator_feed > 100) {
-    input_elevator_feed = 0.1066;
+    input_elevator_feed = 100;
   }
-  return input_elevator_feed * 0.1066 / 100;
+
+  //  double correctedDuration = FEED_CONSTANT * (1 / ( 0.0000356 * pow(input_elevator_feed,2) + 0.00644 * input_elevator_feed));
+  //  double correctedDuration = FEED_CONSTANT * (1/(-0.00000130911*pow(input_elevator_feed,3)+0.000220708*pow(input_elevator_feed,2)+0.0010203*input_elevator_feed));
+  double correctedDuration = FEED_CONSTANT * (3 / (-0.00000010682 * pow(input_elevator_feed, 4) + 0.0000215 * pow(input_elevator_feed, 3) - 0.00134 * pow(input_elevator_feed, 2) + 0.06 * input_elevator_feed - 0.418));
+  // double correctedDuration = FEED_CONSTANT;
+  UART_PORT.print("correctedDuration: ");
```

Comment:
SCIENCE prefix missing

Reply:
It wasn't meant to be picked up by ROS, it's just for debugging.